test/array.test.js

```diff
@@ -2,13 +2,31 @@ const favicons = require("../src");
 const test = require("ava");
 const fs = require("fs");
 
-const { logo_png, logo_svg, normalize } = require("./util");
+const { snapshotManager } = require("ava/lib/concordance-options");
+const { factory } = require("concordance-comparator");
+const Fiber = require("fibers");
+
+const { logo_png, logo_svg } = require("./util");
+const { Image } = require("./Image");
+
+snapshotManager.plugins.push(factory(Image, v => new Image(v[0], v[1])));
 
 test("should accept an array of either buffers or paths to source images", async t => {
   t.plan(1);
 
   // eslint-disable-next-line no-sync
   const result = await favicons([logo_png, fs.readFileSync(logo_svg)]);
 
-  t.snapshot(normalize(result));
+  for (const image of result.images) {
+    // '.ico' is not supported by resemble
+    if (image.name.endsWith(".ico")) delete image.contents;
```

Comment:
If `resemble` can't handle `.ico` files, I'm afraid it's not a good choice for us, unless we find an alternative way to make sure these files don't break.

Reply:
I'll make `.ico` fallback to binary compare.